andela/technites-bn-frontend
# #168049977,#168049978 Trips stats and most visited destintations

#### What does this PR do?
 - Add two components to the dashboard
 1). Getting the most visited destination
 2). Getting the trip statistics
#### Description of Task to be completed?
#### How should this be manually tested?
 - login and head to `/dashboard`
 - user will see both components and interact with them.
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#168049977](https://www.pivotaltracker.com/story/show/168049977)
[#168049978](https://www.pivotaltracker.com/story/show/168049978)
#### Screenshots (if appropriate)
#### Questions:
N/A
